.circleci/config.yml

```diff
@@ -205,6 +218,12 @@ jobs:
           name: Run tests
           command: bundle exec fastlane test_tvos
           no_output_timeout: 30m
+      - when:
+          condition: << pipeline.parameters.generate_snapshots >>
+          steps:
+            - run:
+                name: Run create_snapshot_pr 
+                command: bundle exec fastlane create_snapshot_pr version:"tvos"
```

Comment:
We don't actually need this one. `tvOS` tests use the equivalent version (so tvOS 15 uses iOS 15 snapshots). Could it be removed?